encrypted-config-value/src/main/java/com/palantir/config/crypto/algorithm/AesAlgorithm.java

```diff
@@ -40,7 +44,7 @@
     public static final String ALGORITHM_TYPE = "AES";
 
     private static final int GCM_AUTH_TAG_LENGTH = 128;
-    private static final int IV_LENGTH = 256 / Byte.SIZE;
+    private static final int IV_LENGTH = 96 / Byte.SIZE;
```

Comment:
Yep, will fix up the units.

I am pretty against making these configurable at all. There is AFAIK zero reason to not use 12 byte IVs and 16 byte tags.